app/constants/DR/buttonStyles.js

```diff
@@ -0,0 +1,26 @@
+import {
```

Comment:
Why we should place in thee constants folder? 
Could we add the style near to the components?

Reply:
this file are global styles, all the buttons in the project should have  this style

Reply:
They're not for a specific one